packages/mos-gateway/src/coreHandler.ts

```diff
@@ -725,7 +725,7 @@ export class CoreHandler {
 	executeFunction(cmd: PeripheralDeviceCommand, fcnObject: CoreHandler | CoreMosDeviceHandler): void {
 		if (cmd) {
 			if (this._executedFunctions[unprotectString(cmd._id)]) return // prevent it from running multiple times
-			this.logger.debug(cmd.functionName, cmd.args)
+			this.logger.debug(cmd.functionName || cmd.actionId || '', cmd.args)
```

Comment:
Hm yes, the actions are really only implemented by the subdevices of the playout gateway for now. I still think it's useful to have this log though because it will give a trace in the logs as to why the method wasn't found on the fcnObject.